src/firefox/remote.js

```diff
@@ -4,15 +4,53 @@ import {RemoteTempInstallNotSupported, WebExtError} from '../errors';
 import defaultFirefoxConnector from 'node-firefox-connect';
 
 const log = createLogger(__filename);
+
+
+// Flow Types
+
+import type FirefoxClient from 'firefox-client';
+
+export type FirefoxConnectorFn =
+  (port?: number) => Promise<FirefoxClient>;
+
+export type ConnectExtraParams = {
+  connectToFirefox: FirefoxConnectorFn,
+};
+
+export type FirefoxRDPAddonActor = {
+  id: string,
+  actor: string,
+};
+
+export type FirefoxRDPResponseError = {
+  error: {
+    message: string,
+  },
+};
+
+export type FirefoxRDPResponseAny = Object;
+
+export type FirefoxRDPResponseAddon = {
+  addon: FirefoxRDPAddonActor,
+};
+
+export type FirefoxRDPResponseMaybe =
+  FirefoxRDPResponseError | FirefoxRDPResponseAddon | FirefoxRDPResponseAny;
```

Comment:
It looks like `FirefoxRDPResponseAny` could be replaced with `Object`. However, if the last check is `Object`, does it matter to specify any other object types? It's nice to document the error and add-on response types though.